config/index.js

```diff
@@ -1,18 +1,30 @@
-const config = {
-  messages: {
-    enter: 'bom dia a todos menos a um',
-    exit: 'XAU AI _BORDAS_',
-  },
-
-  activities: {
-    enter: 'Pepe as a Service',
-    exit: `Screw u guys I'm going home`,
-  },
-
-  botToken: process.env.BOT_TOKEN || '',
-  mainChannelId: process.env.MAIN_CHANNEL_ID || '',
-  msgThrottleTime: process.env.MSG_THROTTLE_TIME || 30,
-  msgRetryThreshold: process.env.MSG_RETRY_THRESHOLD || 4,
-};
+const fs = require('fs');
+
+const modules = {};
+
+function config(configPath) {
+  if (!configPath) {
+    return undefined;
+  }
+
+  const [fileName, ...property] = configPath.split('.');
+
+  return modules[fileName][property];
+}
+
+function loadModules(modulePath) {
+  const files =fs.readdirSync(modulePath);
+  const filesToLoad = files.filter(file => !file.startsWith('index'));
+
+  filesToLoad.forEach(file => {
+    const [fileName] = file.split('.');
+
+    // eslint-disable-next-line import/no-dynamic-require
```

Comment:
What is this witchcraft?

Reply:
Tells ESLint to look away on the next line.
By default ESLint's rules squawks when we use `require` elsewhere other than the top of file.